diff --git a/matter_server/server/const.py b/matter_server/server/const.py
index 1716868..8c913d9 100644
--- a/matter_server/server/const.py
+++ b/matter_server/server/const.py
@@ -5,14 +5,4 @@ from typing import Final
 # The minimum schema version (of a client) the server can support
 MIN_SCHEMA_VERSION = 2
 
-# the paa-root-certs path is hardcoded in the sdk at this time
-# and always uses the development subfolder
-# regardless of anything you pass into instantiating the controller
-# revisit this once matter 1.1 is released
-PAA_ROOT_CERTS_DIR: Final[pathlib.Path] = (
-    pathlib.Path(__file__)
-    .parent.resolve()
-    .parent.resolve()
-    .parent.resolve()
-    .joinpath("credentials/development/paa-root-certs")
-)
+PAA_ROOT_CERTS_DIR: Final[pathlib.Path] = pathlib.Path("@paacerts@")
diff --git a/matter_server/server/device_controller.py b/matter_server/server/device_controller.py
index 1ef9163..55325b8 100644
--- a/matter_server/server/device_controller.py
+++ b/matter_server/server/device_controller.py
@@ -88,9 +88,6 @@ class MatterDeviceController:
 
     async def initialize(self) -> None:
         """Async initialize of controller."""
-        # (re)fetch all PAA certificates once at startup
-        # NOTE: this must be done before initializing the controller
-        await fetch_certificates()
         # Instantiate the underlying ChipDeviceController instance on the Fabric
         self.chip_controller = self.server.stack.fabric_admin.NewController(
             paaTrustStorePath=str(PAA_ROOT_CERTS_DIR)
@@ -158,9 +155,6 @@ class MatterDeviceController:
         if self.chip_controller is None:
             raise RuntimeError("Device Controller not initialized.")
 
-        # perform a quick delta sync of certificates to make sure
-        # we have the latest paa root certs
-        await fetch_certificates()
         node_id = self._get_next_node_id()
 
         success = await self._call_sdk(
@@ -201,12 +195,6 @@ class MatterDeviceController:
         if self.chip_controller is None:
             raise RuntimeError("Device Controller not initialized.")
 
-        # perform a quick delta sync of certificates to make sure
-        # we have the latest paa root certs
-        # NOTE: Its not very clear if the newly fetched certificates can be used without
-        # restarting the device controller
-        await fetch_certificates()
-
         node_id = self._get_next_node_id()
 
         success = await self._call_sdk(
